client/components/domains/example-domain-suggestions/index.jsx

```diff
@@ -2,37 +2,109 @@
+					<svg width="295" height="102" viewBox="0 0 295 102" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think something went wrong when formatting this SVG, I get this error when visiting the domains step in NUX:
```
Error: <path> attribute d: Expected number, "….828 3.326-2.183h-1.26zm6.138-.9…".
```
And visually it looks broken too:
<img width="1035" alt="screen shot 2017-06-22 at 18 15 31" src="https://user-images.githubusercontent.com/3392497/27444537-f35d8f4a-5776-11e7-878c-80b4995a0c03.png">